### DISCUSSION TITLE
CRtouch difference between stock and professional

### DISCUSSION THREAD

**Discussion by ghost** [mriscoc/Ender3V2S1#604] [Ideas]
So I upgraded my S1 from stock to professional and I have some issues with stability of the CR touch.
So I did some searching and found out that this happens more often.

So what I noticed when doing any bed leveling is that the stock firmware actually does it different than the prof.
When using multiple probes, the prof firmware will retract the pin between every probe, the stock firmware does not do this (in fact it does not retract the pin during the whole levelling session).
You can see the behaviour of the stock firmware here https://youtu.be/g4aRqfSn7QU?t=143

I was wondering if this might account for the problems people are experiencing.

**Comment by mriscoc** [comment]
For small meshes you can enable HS in Advanced/Probe Settings
https://github.com/mriscoc/Ender3V2S1/wiki/3D-BLTouch#notes-about-crtouch
